packages/components/src/components/checkbox/checkbox.lite.tsx

```diff
@@ -29,12 +28,11 @@ export default function DBCheckbox(props: DBCheckboxProps) {
 	// jscpd:ignore-start
 	const state = useStore<DBCheckboxState>({
 		initialized: false,
-		_id: DEFAULT_ID,
-		_messageId: DEFAULT_ID + DEFAULT_MESSAGE_ID_SUFFIX,
-		_validMessageId: DEFAULT_ID + DEFAULT_VALID_MESSAGE_ID_SUFFIX,
-		_invalidMessageId: DEFAULT_ID + DEFAULT_INVALID_MESSAGE_ID_SUFFIX,
-		_descByIds: '',
-
+		_id: 'checkbox-' + uuid(),
+		_messageId: this._id + DEFAULT_MESSAGE_ID_SUFFIX,
+		_validMessageId: this._id + DEFAULT_VALID_MESSAGE_ID_SUFFIX,
+		_invalidMessageId: this._id + DEFAULT_INVALID_MESSAGE_ID_SUFFIX,
+		_descByIds: `${this._messageId}`,
```

Comment:
```suggestion
		_descByIds: this._messageId,
```

why couldn't this be a simple assignment?

Reply:
https://github.com/db-ui/mono/pull/2599#discussion_r1596343113